### PR TITLE
qmp_command:add 'query-stats' and 'query-stats-schema' commands

### PR DESCRIPTION
**Pull request by yiqianwei**
ID: 2151750

Signed-off-by: Yiqian Wei <yiwei@redhat.com>

### CODE REVIEW

**Comment by yiqianwei** [comment]
Test results:
(1/1) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qmp_command.qmp_query-stats_vm.q35: PASS (396.35 s)

(1/1) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qmp_command.qmp_query-stats_vcpu.q35: PASS (707.93 s)

(1/1) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qmp_command.qmp_query-stats-schemas.q35: PASS (410.73 s)

**Comment by yiqianwei** [comment]
@jingzhao84  and @yanan-fu , Please help to review,thanks.

**Comment by yanan-fu** [comment]
@yiqianwei  Call for id, thanks!

File: qemu/tests/cfg/qmp_command.cfg [hunk 1]
```diff
@@ -203,3 +203,15 @@
...
+            cmd_result_check = "contain"
+            cmd_return_value = "['provider', 'stats', 'name', 'value']"
+        - qmp_query-stats_vcpu:
+            qmp_cmd = "query-stats target=vcpu"
+            cmd_result_check = "contain"
+            cmd_return_value = "['provider', 'stats', 'name', 'value', 'qom-path']"
+        - qmp_query-stats-schemas:
+            qmp_cmd = "query-stats-schemas"
+            cmd_result_check = "contain"
+            cmd_return_value = "['provider', 'stats', 'name', 'exponent', 'type', 'unit', 'target']"
```
Comment (yanan-fu) [inline qemu/tests/cfg/qmp_command.cfg:217]: Do we to add `base` ?

Comment (yiqianwei) [reply qemu/tests/cfg/qmp_command.cfg:217]: ok, we need add 'base'. thanks

Comment (yiqianwei) [reply qemu/tests/cfg/qmp_command.cfg]: @yanan-fu , I have updated it, please check it, thanks

**Comment by yiqianwei** [comment]
Test results:

 (1/3) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qmp_command.qmp_query-stats_vm.q35: PASS (66.04 s)

 (2/3) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qmp_command.qmp_query-stats_vcpu.q35: PASS (92.32 s)

 (3/3) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qmp_command.qmp_query-stats-schemas.q35: PASS (88.12 s)

**Review by yanan-fu — APPROVED** [review]
Ack

**Pull request merged**